pages/dkp/kommander/2.3/clusters/custom-domain-certificate/managed/index.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: layout.pug
+navigationTitle: Managed Cluster 
+title: Customize a domain and certificate in a Managed Cluster
+menuWeight: 20
+excerpt: Configure a custom domain and certificate in a Managed or Attached Cluster
+beta: false
+enterprise: false
+---
+
+This section describes how to enable custom domains and certificates in your **managed and attached** clusters. Refer to the [Customize a domain and certificate in the Management Cluster][management] documentation, if you want to set up custom domains and certificates for your **Management Cluster** during the installation of DKP.
```

Comment:
This page applies to all clusters, including the Management Cluster

Reply:
```suggestion
This section describes how to enable custom domains and certificates in your **Management and any Managed or Attached** clusters after the installation of DKP. If you want to set up custom domains and certificates for your **Management Cluster** during installation, refer to the [Customize a domain and certificate during installation][management] documentation.
```